config.yml

```diff
@@ -290,11 +304,11 @@ scoutsuite:
       - scout --version
 
 serverless:
-  "1.48":
+  "1.51":
```

Comment:
You need to update `env: IMAGE=serverless VERSION=1.48` in `.travis.yml`: `matrix.include`

Reply:
It's the matrix Travis use to know which job to run:

![image](https://user-images.githubusercontent.com/3366767/64294010-2247e500-cf6e-11e9-8007-393cbf217807.png)